File: C++/Programming Basics with C++/10. For-loop/simple_tasks/LeftAndRightSum.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    int n;
    cin >> n;
    int leftSum = 0;
    int rightSum = 0;
    for (int i = 0; i < n; i++) {
        int num;
        cin >> num;
        leftSum += num;
    }
    for (int i = 0; i < n; i++) {
        int num;
        cin >> num;
        rightSum += num;
    }
    if (leftSum == rightSum) {
        cout << "Yes, sum = " << leftSum << endl;
    } else {
        cout << "No, diff = " << abs(leftSum - rightSum) << endl;
    }
    return 0;
}

File: C#/Programming Basics with C#/10. For-Loop/06. Left and Right Sum/LeftandRightSum.cs
using System;

class LeftandRightSum
{
    static void Main()
    {
        int n = int.Parse(Console.ReadLine());
        int sum1 = 0;
        int sum2 = 0;
        for (int i = 1; i <= n; i++)
        {
            int num = int.Parse(Console.ReadLine());
            sum1 = sum1 + num;
        }
        for (int i = 1; i <= n; i++)
        {
            int num = int.Parse(Console.ReadLine());
            sum2 = sum2 + num;
        }
        if (sum1 == sum2)
        {
            Console.WriteLine($"Yes, sum = {sum1}");
        }
        else
        {
            Console.WriteLine($"No, diff = {Math.Abs(sum1 - sum2)}");
        }
    }
}

